aries-site/src/pages/components/index.js

```diff
@@ -40,7 +40,19 @@ const Components = () => (
           </Paragraph>
         </Box>
       </PageIntro>
-      <CardGrid cards={cards} />
+      <ContentSection>
+        <Subsection name="Layouts">
```

Comment:
Really like having these h2s on the page. helps a lot to feel less overwhelmed by cards